draftfast/constants/__init__.py

```diff
@@ -0,0 +1,2 @@
+from roster_size import ROSTER_SIZE_BY_SITE_BY_SPORT
+from positions import POSITIONS_BY_SITE_BY_LEAGUE
```

Comment:
I think these also need `assert`s to pass the build

Reply:
i guess i should have just run the linting locally!